Resources/Storage/test/Test_FilePlugin.py

```diff
@@ -153,7 +154,7 @@ def localPutFile( fn, size = 0 ):
     # wrong size
     res = localPutFile( self.existingFile, size = -1 )
     self.assert_( res['OK'], res )
-    self.assert_( self.existingFile in res['Value']['Failed'] )
+    self.assert_( self.existingFile in res['Value']['Failed'], res )
```

Comment:
assert_ form of unittest assertion is declared obsoleted in the unittest documentation. Use assertEqual or other assertXXX methods